storage/client_range_tree_test.go

```diff
@@ -226,11 +208,22 @@ func TestInsert(t *testing.T) {
 		"t", "u", "v", "w", "x", "y", "z",
 	}
 
-	for _, key := range keys {
+	for _, key := range keysInsert {
 		if err := db.AdminSplit(key); err != nil {
 			t.Fatal(err)
 		}
 		tree, nodes := loadTree(t, db)
 		VerifyTree(t, tree, nodes, key)
 	}
+
+	// To test merging, we just call merged on the lowest key to merge all
```

Comment:
hmm.  I know I wrote AdminMerge on this.  I maybe reverted it accidentally.  Anyway, fixed.